lib/tasks/load_tags_from_content_api.rake

```diff
@@ -0,0 +1,39 @@
+desc "Load tags from Content API into Collections Publisher."
+task :load_tags_from_content_api => :environment do
+  require 'gds_api/content_api'
+
+  content_api_endpoint = Plek.new.find("content_api")
+  content_api = GdsApi::ContentApi.new(content_api_endpoint)
+
+  puts "There are currently #{Tag.count} tags in the database"
+  puts "Fetching all tags from the Content API..."
+  tags = content_api.get_list!("#{content_api_endpoint}/tags.json")
+
+  puts "Received a total of #{tags.count} tags"
+  puts "Loading tags into Collections Publisher..."
+  tags.each do |api_tag|
+    unless Tag.where(slug: api_tag.slug).any?
+      tag = Tag.new(slug: api_tag.slug,
+                    description: api_tag.details.description,
+                    title: api_tag.title)
+
+      tag.save!
+    end
+  end
+
+  # On the first run of this task, the tags might not exist in the database.
+  # Traverse through all fetched tags again and assign parents accodingly.
+  puts "Assigning child tags to parents..."
+  tags.each do |api_tag|
+    existing_tag = Tag.where(slug: api_tag.slug).first
+
+    if existing_tag.parent.nil? and not api_tag.parent.nil?
+      parent = Tag.where(slug: api_tag.parent.slug).first
+      existing_tag.parent = parent
+      existing_tag.save!
+    end
+  end
+
+  puts "There are now #{Tag.count} tags in the database"
```

Comment:
Hopefully this will equal the number of tags in panopticon.  I can imagine there are some in collections-publisher but not in panopticon already, though, so perhaps we should explicitly check that there are no excess tags.